src/ExtInterface/ReefMod/scenarios.jl

```diff
@@ -4,35 +4,8 @@
 Run ADRIA coral ecosystem model with ReefMod data.
 """
 function run_scenarios(::Type{ReefModDomain}, scens::DataFrame, dom::ReefModDomain, rcp::String)
```

Comment:
Hi Rose,

On reviewing what's left of the code, I think we don't need this entire function anymore?

It looks like we can call `run_scenarios()` as normal and it'll just work

Reply:
Sure, will do

Reply:
Removed the scenarios.jl file

Reply:
So I tried loading the Domain and running scenarios.  Domain runs fine but there's a dimension mismatch when the scenarios are run so I'll look into fixing that before this is merged.